libs/langchain/langchain/llms/anthropic.py

```diff
@@ -23,9 +23,13 @@ class _AnthropicCommon(BaseLanguageModel):
     async_client: Any = None  #: :meta private:
     model: str = "claude-2"
     """Model name to use."""
+    model_name: str = "claude-2"
```

Comment:
using pydantic you can declare an alias

Reply:
If I'm not mistaken, you can use a pydantic field's alias during but not after instantiation.

```python
>>> from pydantic import BaseModel, Field
>>>                                                                    
>>> class TestModel(BaseModel):                                        
...     model: str = Field(default='default_model', alias='model_name')
...                                                                    
>>>                                                                    
>>> test = TestModel(model_name='test_model')                          
>>> test.model                                                         
'test_model'
>>> test.model_name
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/appuser/.local/lib/python3.11/site-packages/pydantic/main.py", line 726, in __getattr__
    raise AttributeError(f'{type(self).__name__!r} object has no attribute {item!r}')
AttributeError: 'TestModel' object has no attribute 'model_name'. Did you mean: 'model_dump'?
```

This is why I took a seemingly roundabout approach instead.

Reply:
is it important to use it after instantiation?

Reply:
Maybe not. It just seems to enable greater consistency. It's kind of a "gotcha" right now. I can imagine if you have a list of llms and you want to display (model_name, result) pairs, it may be nice for them to use the same variable/property names. I've ran into that issue and I resolved it by checking the type of model. Perhaps that was the motivation behind the issue this PR responds to. 

That said, this PR achieves the goal in a suboptimal way. I think the best solution may be to use `model_name` and `max_tokens` across all models and then do what's needed under the hood to make the calls. Then again, that may not be a one-size-fits-all solution, and it obscures the underlying api. Maybe a field alias and a `get_model_name` method would be better. 

Personally, I like to compare results from openai, anthropic, and anyscale. The different variable names have been a minor inconvenience that greater abstractions like langsmith are making less painful.

Reply:
yeah if we want to enforce consistent naming, we should probably do that more stricly with a get_model_name or something. that is probably a separate thing from field alias

im personally more disposed to add field alias at the moment (will probably be easier/lower lift/less controversial)

Reply:
Agreed. Later tonight, I will revise to use a simple alias. Thank you for the guidance. :)